src/Pages/Profile/Settings.tsx

```diff
@@ -36,50 +38,22 @@ const Settings: React.FC = () => {
 
   return (
     <ProfilePageLayout>
-      <Title
-        level={2}
-        style={{
-          display: 'inline',
-          position: 'absolute',
-          marginTop: '12vh',
-          marginLeft: '5vw',
-        }}
-      >
-        {t('settings.title')}
-      </Title>
-      <Form style={{ position: 'absolute', marginTop: '20vh', width: '100vw' }}>
-        <Form.Item label={t('settings.languageLabel')}>
-          <Cascader
-            placeholder=" "
-            cancelText="Cancel"
-            confirmText="Confirm"
-            options={LANGUAGE_OPTIONS}
-            visible={visible}
-            onSelect={onSelectLanguage}
-            onClose={() => {
-              setVisible(false);
-            }}
-          />
-          <Typography
-            style={{
-              position: 'absolute',
-              right: '25vw',
-              display: 'inline',
-              top: '2vh',
-            }}
-          >
-            {language}
-          </Typography>
-          <Button
-            style={{ position: 'absolute', right: '5vw', top: '0.7vh' }}
-            onClick={() => {
-              setVisible(true);
-            }}
-          >
-            <ArrowRightOutlined />
-          </Button>
-        </Form.Item>
-      </Form>
+      <div className="section">
+        <h4 className="title title-h4">{t('settings.title')}</h4>
+        <div className="spacer-regular"></div>
+        <Form className="portato-form">
+          <div className="input-wrapper">
+            <label>{t('settings.languageLabel')}</label>
+            <Selector
```

Comment:
This selector should default to the currently selected language and always have 1 of the options active. Deselecting all buttons should be impossible.